### ISSUE TITLE
Consider including a port of space Cadet pinball 

### ISSUE THREAD

**Issue by weirdbeardgame** [rozniak/xfce-winxp-tc#267]
https://github.com/k4zmu2a/SpaceCadetPinball - Space Cadet pinball... Nough said. But seeing you did your own notepad app, I feel this would be a great inclusion to the overall experience

**Comment by rozniak** [comment]
Yeah, I'll include it eventually, most likely as a patched version to drop ImGui to have it better integrated with the UI.
